Feature branch

marchelo271 opened the pull request:
** Description of project**
Added linters file to the project
Added .html file
Added .css file
Added test.md to gitignore

Removed the hint report file
Updated the Readme like the format proposed by microverseFirst pull request

marchelo271 committed:
checking development branch

marchelo271 committed:
checking feature branch

marchelo271 committed:
adding linters config

marchelo271 committed:
git ignore file added

marchelo271 committed:
added .Hintrc file

marchelo271 committed:
added stylent file

marchelo271 committed:
updating  html file Hello-microverse

marchelo271 committed:
adding css file

marchelo271 committed:
add test.md file and adding to gitignore

marchelo271 committed:
Updating Readme file for presentation

aimalamiri commented on readme.md:
```diff
@@ -0,0 +1,7 @@
+
+This project should demostrate git and github skills including the linters and 
+showing an efficient handle on how to use github Flow. 
+
```
Good job, but it's highly recommended to use this [template](https://github.com/microverseinc/readme-template) by Microverse to better organizer your repository.

aimalamiri reviewed (changes requested):
Hi @marchelo271, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![AlmostThereReachGIF](https://user-images.githubusercontent.com/34779928/191347504-9f70659c-a01f-48a1-ad46-b8722d46ffca.gif)

## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
Good job! for implementing most of the requirements, but please make sure to consider the remaining as well.
1. The linters are not properly configured, please make sure to add the content to your `.github/workflows/linters.yml` file.
2. It's highly recommended to keep your repository as clean as possible by avoiding unnecessary files. In this repository I see a folder called `hint-report` please make sure to remove it because it's not part of the project.
3. Please summarize the task in your pull request summary, so the reviewer can have an idea about the pull request.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

marchelo271 committed:
removing hint report

marchelo271 committed:
Delete hint report

marchelo271 committed:
updating .github for linters setup

marchelo271 committed:
Merge branch 'feature-branch' of https://github.com/marchelo271/hello_microverse into feature-branch

marchelo271 commented:
Added linters file to the project
Added .html file 
Added .css file
Added test.md to gitignore

Removed the hint report file 
Updated the Readme like the format proposed by microverse

marchelo271 committed:
updating readme file

abdulhamiid commented on readme.md:
```diff
@@ -0,0 +1,65 @@
+🤝 Contributing
+⭐️ Show your support
+🙏 Acknowledgements
+❓ FAQ
```
- [ ] As requested by the previous reviewer, please let us copy and customize the contents of this [README template](https://github.com/microverseinc/readme-template) accordingly.

- [ ] The purpose of README is to introduce and explain a project. It contains information that is commonly required to understand what the project is about. so please update your readme by providing other people with information about your project 😊

abdulhamiid commented on styles.css:
```diff
@@ -0,0 +1,3 @@
+.h1{
+    color: blue;
+}
```
- [ ] Your design 👇 

![hello](https://user-images.githubusercontent.com/78251927/205367436-bdcda8b0-eea4-4a0e-a0a8-297c84ff2877.PNG)

**[OPTIONAL]** As this requirement was not mentioned by the previous reviewer, it is marked optional, but recommended. please kindly consider making the text color effective. I see that you used `.h1` in your CSS file instead of using just `h1` selector. as your `h1` tag does not have `class=h1` in it. 😃
```
<h1> Hello Microverse </h1>
```

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @marchelo271 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Some linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

### Linters Checks 👇 
![linterrr](https://user-images.githubusercontent.com/78251927/205361452-07f5f447-876a-495f-80a1-239e60253649.PNG)

- [ ] As we can see that the `Linters/Stylelint` checks are failing ❌,  whereas, all linter results should be green ✔️. AKA: Linter should report NO errors.

- [ ] To check the `Linters/Stylelint`  error which is making the checks to fail, 
- [ ] Run this command in your Editor terminal👇 
```
npx stylelint "**/*.{css,scss}"
```
- [ ] All the `Stylelint` errors will appear and you can fix them manually or by using the auto-fix command.
- [ ] To use auto Fix, Run 👇 
```
npx stylelint "**/*.{css,scss}" --fix
```

### Pull request summary

![image](https://user-images.githubusercontent.com/78251927/205374063-48659151-1ed9-49ca-9e3c-fce62a09704e.png)

- [ ] As requested by the previous reviewer, please add your pull request summary here ☝️, and not in the comment section as you did. 😊 

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

marchelo271 committed:
fixing errors with linters in css file

huseyinbiyik commented on readme.md:
```diff
@@ -0,0 +1,65 @@
+[Hello-microverse] is a...
+
+🛠 Built With
+Html and CSS
+
+
+Client
+Server
+Database
+Key Features
+
+[html]
+[css]
+[linters]
+
+
+
+
+💻 Getting Started
+This project can be used for a first learner of github environment
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder
+
+
+👥 Authors
+Erick Peñaloza
+
+
+⭐️ Show your support
+This is an effort for start a career in development world
+
+🙏 Acknowledgments
+Do not give up, continue with hard working 
+
```
As the previous reviewer requested, please use this [format](https://github.com/microverseinc/readme-template) for your Readme file. Right now it looks like below.
<img width="917" alt="Screenshot 2022-12-03 at 2 18 35 AM" src="https://user-images.githubusercontent.com/25887253/205405886-85364976-a3ac-48d4-bf6d-2e69624e3486.png">

huseyinbiyik reviewed (changes requested):
Hi @marchelo271,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![l](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)


 ## Highlights
- Linters are passing 🟢

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] Please use descriptive names for your branches. Instead of "feature" you can use "setup". [Click](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#:~:text=Remember%20that%20your%20branch%20name%20should%20be%20descriptive%20(e.g.%3A%20refactor%2Dauthentication%2C%20user%2Dcontent%2Dcache%2Dkey%2C%20make%2Dretina%2Davatars)%2C%20so%20that%20others%20can%20see%20what%20is%20being%20worked%20on.) to learn more about it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

marchelo271 committed:
updating readme file according microverse template

marchelo271 committed:
last update for readme file

algerina reviewed (changes requested):
Hi @marchelo271 , 

You've done an amazing job so far fir 👏 💯 👍
There are some issues that you still need to work on to go to the next project but you are almost there! 

## Highlights
  -  You have a descriptive Pull Request 
  -  Linters are passing
  - Your project looks good!

## Required Changes ♻️


- :red_circle:  While you did a great job, your README file still needs some changes to look professional. I recommend that you delete the content of the file and then copy-paste it again from Microverse Template. 

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

marchelo271 committed:
updated readme file according to microverse template

marchelo271 committed:
adding a new README file

ridwanediallo reviewed (approved):
# STATUS: APPROVED 🟢
## Hi @marchelo271 , 
![Congratulations!GIF](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
